src/py/flwr/cli/new/new.py

```diff
@@ -173,9 +165,88 @@ def new(
         "username": username,
     }
 
+    # List of files to render
+    if framework_str == "flwrtune":
+        files = {
+            ".gitignore": {"template": "app/.gitignore.tpl"},
+            "pyproject.toml": {"template": f"app/pyproject.{framework_str}.toml.tpl"},
+            "README.md": {"template": f"app/README.{framework_str}.md.tpl"},
+            f"{import_name}/__init__.py": {"template": "app/code/__init__.py.tpl"},
+            f"{import_name}/server.py": {"template": "app/code/flwrtune/server.py.tpl"},
+            f"{import_name}/client.py": {"template": "app/code/flwrtune/client.py.tpl"},
+            f"{import_name}/app.py": {"template": "app/code/flwrtune/app.py.tpl"},
+            f"{import_name}/models.py": {"template": "app/code/flwrtune/models.py.tpl"},
+            f"{import_name}/dataset.py": {
+                "template": "app/code/flwrtune/dataset.py.tpl"
+            },
+            f"{import_name}/conf/config.yaml": {
+                "template": "app/code/flwrtune/config.yaml.tpl"
+            },
+            f"{import_name}/conf/static_config.yaml": {
+                "template": "app/code/flwrtune/static_config.yaml.tpl"
+            },
+        }
+
+        # Task specific context
+        context["llm_task_str"] = llm_task_str
+        context["fraction_fit"] = "0.2" if llm_task_str == "code" else "0.1"
+        context["task_name"] = (
+            "General NLP"
+            if llm_task_str == "generalnlp"
+            else (
+                "Finance"
+                if llm_task_str == "finance"
+                else "Medical" if llm_task_str == "medical" else "Code"
+            )
+        )
+        context["num_clients"] = (
+            "20"
+            if llm_task_str in ("generalnlp", "medical")
+            else "50" if llm_task_str == "finance" else "10"
+        )
+        context["dataset_name"] = (
+            "vicgalle/alpaca-gpt4"
+            if llm_task_str == "generalnlp"
+            else (
+                "FinGPT/fingpt-sentiment-train"
+                if llm_task_str == "finance"
+                else (
+                    "medalpaca/medical_meadow_medical_flashcards"
+                    if llm_task_str == "medical"
+                    else "lucasmccabe-lmi/CodeAlpaca-20k"
+                )
+            )
+        )
```

Comment:
I wonder if this could be generated from a template as well... If we add support for new templates, this logic will need to be always updated.